### PR TITLE
Igal/day2 flow fix

### PR DESCRIPTION
**Pull request by tsorya**
Fixing day2 flow 
1. Upgrade oc client version
2. FIx kubeconfig path after it was changed by 7e043434d4900060827887a06474a6563c931bf5

### CODE REVIEW

File: scripts/install_minikube.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ function install_oc() {
     if ! [ -x "$(command -v oc)" ]; then
         echo "Installing oc..."
         for i in {1..4}; do
-            curl --retry 3 -SL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.6.0/openshift-client-linux-4.6.0.tar.gz | tar -xz -C /usr/local/bin && break
+            curl --retry 3 -SL curl --retry 3 -SL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.8.0-rc.0/openshift-client-linux-4.8.0-rc.0.tar.gz | tar -xz -C /usr/local/bin && break
```
Comment (YuviGold) [inline scripts/install_minikube.sh:39]: this doesn't look good.

**Review by YuviGold — COMMENTED** [review]
Would @eliorerz PR https://github.com/openshift/assisted-test-infra/pull/915 solve it for you?

File: scripts/install_minikube.sh (hunk 1 above)
Comment (tsorya) [reply scripts/install_minikube.sh:39]: what does it mean?

**Comment by tsorya** [comment]
> Would @eliorerz PR #915 solve it for you?

nope, only one line

**Review by YuviGold — APPROVED** [review]
LGTM

**Pull request merged**